docroot/modules/humsci/hs_admin/hs_admin.info.yml

```
@@ -4,3 +4,4 @@ description: "A module that modifies adminstration pages for improved UX"
 core_version_requirement: ^10.3 || ^11
 version: "1.0"
 package: 'Humanities & Sciences'
+dependencies: [ckeditor5_plugin_pack:ckeditor5_plugin_pack_bookmark]
```

For consistency, let's change how the list is defined:

```suggestion
dependencies: 
   - ckeditor5_plugin_pack:ckeditor5_plugin_pack_bookmark
```